packages/mesos/build

```diff
@@ -56,6 +56,18 @@ disk_resource_script="$PKG_PATH/bin/make_disk_resources.py"
 cp /pkg/extra/make_disk_resources.py "$disk_resource_script"
 chmod +x "$disk_resource_script"
 
+# Copy the log-redirection wrapper script for the master/agent.
```

Comment:
guessing here, but can exec -a help us?

http://stackoverflow.com/questions/31747247/change-real-process-name-in-c-on-linux